app/src/heartbeat/schemas/logged_heartbeats.fbs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace com.google.firebase.heartbeat.cpp; // TODO - pick namespace
```

Comment:
Let me know if you have any thoughts on what would be a good namespace. I saw a com.google... namespace for some other fbs file in the repository, but I think that might be an artifact from when firebase was in google3.

Reply:
Updated namespace.